### PR TITLE
VPN-5009 Webext add asserts and tests

### PR DESCRIPTION
**Pull request by strseb**
## Description
grab

### CODE REVIEW

**Review by brizental — APPROVED** [review]
image

File: tests/nativemessaging/helper.cpp [hunk 1]
```diff
@@ -6,29 +6,38 @@
...
   });
-  s_nativeMessagingProcess->start(app, QStringList(),
+  connect(s_nativeMessagingProcess, &QProcess::finished, [&](int exitCode) {
+    s_last_exit_code=exitCode;
+  });
+  s_nativeMessagingProcess->start(app, arguments,
                                   QProcess::Unbuffered | QProcess::ReadWrite);
   if (!s_nativeMessagingProcess->waitForStarted()) {
-    qFatal("Failed to start the naive messaging process");
+    // qFatal("Failed to start the naive messaging process");
```
Comment (Gela) [inline tests/nativemessaging/helper.cpp:32]: Why comment this out?

**Review by mcleinman — COMMENTED** [review]
Same question as Gela about the commented out line, otherwise this looks great to me. (Feel free to merge after addressed.) Thanks for this!

File: tests/nativemessaging/helper.cpp (hunk 1 above)
Comment (strseb) [reply tests/nativemessaging/helper.cpp]: Oops, was debugging exceptions. Removed that :)

**Review by Gela — APPROVED** [review]

**Pull request merged**